src/jit/codegenarm64.cpp

```diff
@@ -9396,6 +9388,38 @@ void                CodeGen::genArm64EmitterUnitTests()
 }
 #endif // defined(DEBUG)
 
+
+//------------------------------------------------------------------------------------------------ //
+// getFirstArgWithStackSlot - returns the first argument with stack slot on the caller's frame.
+//
+// Return value:
+//    The number of the first argument with stack slot on the caller's frame.
+
+unsigned
+CodeGen::getFirstArgWithStackSlot()
+{
+    // Iterate over all the local variables in the lclvartable.
+    // They contain all the implicit argumets - thisPtr, retBuf,
+    // generic context, PInvoke cookie, var arg cookie,no-standard args, etc.
+    unsigned baseVarNum = 0;
+    LclVarDsc* varDsc = nullptr;
+    for (unsigned i = 0; i < compiler->lvaCount; i++)
```

Comment:
I see the implementation of this routine is same as x64 unix.  How about moving this routine to codegencommon.cpp and enabling x64 unix implementation for Arm64?